docs/core/tools/dotnet-tool-update.md

```diff
@@ -15,18 +15,18 @@ ms.date: 02/14/2020
 
 ```dotnetcli
 dotnet tool update <PACKAGE_NAME> <-g|--global>
-    [--configfile] [--framework] [-v|--verbosity]
-    [--add-source]
+    [--configfile <FILE>] [--framework <FRAMEWORK>]
+    [-v|--verbosity <LEVEL>] [--add-source <SOURCE>]
 
-dotnet tool update <PACKAGE_NAME> <--tool-path>
-    [--configfile] [--framework] [-v|--verbosity]
-    [--add-source]
+dotnet tool update <PACKAGE_NAME> <--tool-path <PATH>>
+    [--configfile <FILE>] [--framework <FRAMEWORK>]
+    [-v|--verbosity <LEVEL>] [--add-source <SOURCE>]
 
 dotnet tool update <PACKAGE_NAME>
-    [--configfile] [--framework] [-v|--verbosity]
-    [--add-source]
+    [--configfile <FILE>] [--framework <FRAMEWORK>]
+    [-v|--verbosity <LEVEL>] [--add-source <SOURCE>]
 
-dotnet tool update <-h|--help>
+dotnet tool update -h|--help
```

Comment:
The angle brackets you removed appear on line 18 and 21 in this file, and also in dotnet-tool-uninstall.md. Should they be removed there too?

Reply:
Yes, good catch.